charts/dvpe-deployment-gloo/templates/upstream.yaml

```diff
@@ -0,0 +1,27 @@
+{{- $serviceName := include "service.name" . -}}
+{{- $upstreamName := include "upstream.name" . -}}
+
+{{- if .Values.gloo.enabled }}
+
+apiVersion: gloo.solo.io/v1
+kind: Upstream
+metadata:
+  name: {{ $upstreamName }}
+  namespace: gloo-system
```

Comment:
I would suggest to rather create a parameter gloo.namespace and reference it from all other resource files. This makes it more robust against changes.